### ISSUE TITLE
Vornamen Zeichenbegrenzung

### ISSUE THREAD

**Issue by waweic** [Alovoa/alovoa#163]
Die Länge der Vornamen ist auf 10 Zeichen begrenzt. Hans-Jürgen hat bspw. 11 Zeichen

**Comment by Nonononoki** [comment]
Hi! Sorry about the limit, but it is necessary to not break the layout on small devices. People with longer names usually have a short nickname. Shouldn't "Hans" work in your case?

**Comment by coderamm** [comment]
I don't think this is the most elegant solution - not a UI/UX expert but shouldn't there be a way to rearrange elements irrespective of the lenght? Maybe an adhoc solution/ improvement is make sure the users are aware of the word limit (and maybe your rational behind it) through a pop-up on the sign-up page?
